compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -583,9 +583,7 @@ void addAMDAIEObjectFifoLoweringPasses(OpPassManager &passManager) {
   passManager.addPass(createEraseHALDescriptorTypeFromMemRefPass());
   passManager.addPass(memref::createFoldMemRefAliasOpsPass());
   passManager.addPass(createAMDAIEPackToDmaPass());
-  passManager.addPass(xilinx::air::createCopyToDmaPass());
```

Comment:
In the current implementations, we don't have copy ops used for data movement, so there's no need to keep such pass. In the future, we can implement a new pass to convert copy ops to AMDAIE dma ops if needed.

Reply:
Sorry... I thought you have switched to use pack based tiling strategy for convolution? I can write a pass to convert copy to dma if it's really needed. Let me know what you decide.